boards/arm/nucleo_f303re/nucleo_f303re_defconfig

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM=y
```

Comment:
This symbol is now (logically) deduced from SOC symbols. This is a recent change and assigning it in board's file generates an error at build (as we could see in shippable reports)
Please remove.

Reply:
K